todo/Controllers/CategoriesController.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Todo.Services;
+
+namespace Todo.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class CategoriesController : ControllerBase
+    {
+        private readonly ICategoryService _categoryService;
+
+        public CategoriesController (ICategoryService categoryService)
+        {
+            _categoryService = categoryService;
+        }
+        [HttpGet("categories")]
+        public async Task<IActionResult> GetCategory()
+        {
+            try
+            {
+                return Ok(await _categoryService.GetCategory());
+            }
+            catch
+            {
+                return StatusCode(500);
```

Comment:
```suggestion
                return InternalServerError();
```
nên log lại lỗi nữa nhé, k log lại đến lúc k biết debug kiểu gì đâu